### PR TITLE
Delete user

### LINKED ISSUE #607: Delete account
We should be able to delete a user account completely.

You should do this by adding a new option in our user account menu called _Settings_ and then the on this new settings page, an option Delete my account in red should be available, once the user clicks, a confirm dialog should display a message **Are you sure that you want to delete your account?** should be displayed with a confirm and cancel buttons.

### Why
We should provide an option in our platform to delete accounts, instead of doing this by user request.

### Level
Beginner

### Screenshot attached
Screen Shot 2020-04-07 at 12 53 09 PM

### How to reproduce the user scenario
 - Create a user

### Tips
Please reuse our existing Material-UI components to create the buttons and screens.

### PR DESCRIPTION
**Pull request by spikeboom**
## Description

We should be able to delete a user account completely.

You should do this by adding a new option in our user account menu called Settings and then the on this new settings page, an option Delete my account in red should be available, once the user clicks, a confirm dialog should display a message Are you sure that you want to delete your account? should be displayed with a confirm and cancel buttons.

## Changes

- Where your changes apply

## Issue

> Closes #607

## Impacted Area

> Main page

### CODE REVIEW

File: frontend/src/components/profile/settings.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import { FormattedMessage, injectIntl } from 'react-intl'
+import DeleteUser from './settings/deleteUser'
+
+const styles = theme => ({
+  deleteButton: {
+    backgroundColor: '#F52A2A',
+    color: '#E5E5E5',
+    '&:hover': {
+      backgroundColor: '#F16060'
```
Comment (alexanmtz) [inline frontend/src/components/profile/settings.js:19]: @spikeboom we should use theme colors like:
https://github.com/worknenjoy/gitpay/search?q=theme.&unscoped_q=theme.

Please check the color associate with danger using Material-UI: 
https://material-ui.com/customization/color/

File: modules/users/userDeleteById.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const userDeleteById = async (userParameters) => {
+  try {
+    return await db.sequelize.transaction(async (t) => {
+      const tasks = await models.Task.findAll({
+        where: {
+          userId: userParameters.id
+        }
+      })
+
+      for (const task of tasks) {
```
Comment (alexanmtz) [inline modules/users/userDeleteById.js:14]: Great solution🥇

**Comment by spikeboom** [comment]
I'm still learning to use Git, sorry for the change in frontend/src/translations/br.json

File: frontend/src/components/profile/settings.js (hunk 1 above)
Comment (spikeboom) [reply frontend/src/components/profile/settings.js]: I did not find the color palette for the danger type in the Material UI documentation. So I used the error type. Tell me if I'm right. For any changes I am ready for adjustments.

**Comment by spikeboom** [comment]
and the change in:

console.log('user destroyed', user)
console.log('destroy', user)

I will improve my Git skills

**Comment by alexanmtz** [comment]
@spikeboom no problem, I did these changes just to trigger a new deploy

**Pull request merged**